libtiny_tui/src/tui.rs

```diff
@@ -38,6 +38,28 @@ pub(crate) enum TUIRet {
 const LEFT_ARROW: char = '<';
 const RIGHT_ARROW: char = '>';
 
+/// Command - (name, description, usage)
+struct CmdUsage(&'static str, &'static str, &'static str);
```

Comment:
This unit struct is probably stupid, but at the time I thought it was nice to create all the following constants...

Reply:
I think this is fine, but perhaps we can give names to fields so that the type will become self-documenting. Something like

```rust
struct CmdUsage {
    name: &'static str,
    description: &'static str,
    usage: &'static str,
}
```

Now we can remove the documentation as it's clear from the type.

Reply:
Done!